perfmetrics/scripts/presubmit_test/pr_perf_test/build.sh

```diff
@@ -15,55 +17,83 @@ set -e
+then
+  # Installing requirements
+  echo Installing python3-pip
+  sudo apt-get -y install python3-pip
+  echo Installing libraries to run python script
+  pip install google-cloud
+  pip install google-cloud-vision
+  pip install google-api-python-client
+  pip install prettytable
+  echo Installing fio
+  sudo apt-get install fio -y
+
+  # Executing perf tests for master branch
+  git checkout master
+  echo Mounting gcs bucket for master branch
+  mkdir -p gcs
+  GCSFUSE_FLAGS="--implicit-dirs --max-conns-per-host 100"
+  BUCKET_NAME=presubmit-perf-tests
+  MOUNT_POINT=gcs
+  # The VM will itself exit if the gcsfuse mount fails.
+  CGO_ENABLED=0 go run . $GCSFUSE_FLAGS $BUCKET_NAME $MOUNT_POINT
+  touch result.txt
+  # Running FIO test
+  chmod +x perfmetrics/scripts/presubmit/run_load_test_on_presubmit.sh
+  ./perfmetrics/scripts/presubmit/run_load_test_on_presubmit.sh
+  sudo umount gcs
+
+  # Executing perf tests for PR branch
+  echo checkout PR branch
+  git checkout pr/$KOKORO_GITHUB_PULL_REQUEST_NUMBER
+  echo Mounting gcs bucket from pr branch
+  mkdir -p gcs
+  # The VM will itself exit if the gcsfuse mount fails.
+  CGO_ENABLED=0 go run . $GCSFUSE_FLAGS $BUCKET_NAME $MOUNT_POINT
+  # Running FIO test
+  chmod +x perfmetrics/scripts/presubmit/run_load_test_on_presubmit.sh
+  ./perfmetrics/scripts/presubmit/run_load_test_on_presubmit.sh
+  sudo umount gcs
 
-# Executing integration tests
-GODEBUG=asyncpreemptoff=1 CGO_ENABLED=0 go test ./tools/integration_tests/... -p 1 --integrationTest -v --testbucket=gcsfuse-integration-test -timeout 24m
+  # Show results
+  echo showing results...
+  python3 ./perfmetrics/scripts/presubmit/print_results.py
+fi
+
+# Execute integration tests.
+if [[ "$integrationTestsStr" == *"execute-integration-tests"* ]];
+then
+  echo checkout PR branch
+  git checkout pr/$KOKORO_GITHUB_PULL_REQUEST_NUMBER
 
-# Executing perf tests
-echo Mounting gcs bucket from pr branch
-mkdir -p gcs
-# The VM will itself exit if the gcsfuse mount fails.
-CGO_ENABLED=0 go run . $GCSFUSE_FLAGS $BUCKET_NAME $MOUNT_POINT
-# Running FIO test
-chmod +x perfmetrics/scripts/presubmit/run_load_test_on_presubmit.sh
-./perfmetrics/scripts/presubmit/run_load_test_on_presubmit.sh
-sudo umount gcs
+  # Create bucket for integration tests.
+  # The prefix for the random string
+  bucketPrefix="gcsfuse-integration-test-"
+  # The length of the random string
+  length=5
+  # Generate the random string
+  random_string=$(tr -dc 'a-z0-9' < /dev/urandom | head -c $length)
+  BUCKET_NAME=$bucketPrefix$random_string
+  gcloud alpha storage buckets create gs://$BUCKET_NAME --project=gcs-fuse-test-ml --location=us-west1 --uniform-bucket-level-access
 
-echo showing results...
-python3 ./perfmetrics/scripts/presubmit/print_results.py
+  # Executing integration tests
+  GODEBUG=asyncpreemptoff=1 CGO_ENABLED=0 go test ./tools/integration_tests/... -p 1 --integrationTest -v --testbucket=$BUCKET_NAME -timeout 24m
```

Comment:
Could you make 24m a configurable variable at the top to make it easier to spot and change ?